filebeat/module/haproxy/log/test/httplog-no-headers.log-expected.json

```diff
@@ -2,6 +2,7 @@
     {
         "ecs.version": "1.0.0-beta2",
         "event.dataset": "haproxy.log",
+        "event.duration": 0,
```

Comment:
Is this 0 if it does not exist or is it actually 0 in the log?

Reply:
It's always present in this log format. It's the last 0 of `0/-1/-1/-1/0 ` in the first line, just before the 503 status code.